### PR TITLE
Athena query cost

### PR DESCRIPTION
**Pull request by ialeinikov**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [X] Feature
- [ ] Bug Fix
- [ ] New Query Runner (Data Source)
- [ ] New Alert Destination
- [ ] Other

## Description
Shows a price for an Athena query

Screen Shot 2019-08-17 at 6 31 44 pm
Screen Shot 2019-08-17 at 6 32 49 pm

### CODE REVIEW

File: client/app/pages/queries/query.html [hunk 1]
```diff
@@ -280,6 +280,9 @@ <h3>
             <span class="query-metadata__property" ng-if="queryResult.query_result.data.metadata.data_scanned">Data Scanned
               <strong>{{ queryResult.query_result.data.metadata.data_scanned | prettySize}}</strong>
             </span>
+            <span class="query-metadata__property" ng-if="queryResult.query_result.data.metadata.query_cost">Query cost in USD
```
Comment (arikfr) [inline client/app/pages/queries/query.html]:
```suggestion
            <span class="query-metadata__property" ng-if="queryResult.query_result.data.metadata.query_cost">Cost (USD)
```

File: redash/query_runner/athena.py [hunk 2]
```diff
@@ -123,6 +124,16 @@ def configuration_schema(cls):
             schema['order'].insert(1, 'aws_access_key')
             schema['order'].insert(2, 'aws_secret_key')
 
+        if EXPOSE_COST:
+            schema['order'].append('cost_per_tb')
+            schema['properties'].update({
+                'cost_per_tb': {
+                    'type': 'number',
+                    'title': 'Athena cost per Tb scanned',
```
Comment (arikfr) [inline redash/query_runner/athena.py]:
```suggestion
                    'title': 'Athena cost per Tb scanned (USD)',
```

File: redash/query_runner/athena.py [hunk 3]
```diff
@@ -238,6 +249,9 @@ def run_query(self, query, user):
                     'athena_query_id': athena_query_id
                 }
             }
+            if EXPOSE_COST:
+                price = self.configuration.get('cost_per_tb', 5)
+                data['metadata'].update({'query_cost': '${0:.2f}'.format(price * qbytes * 10e-12)})
```
Comment (arikfr) [inline redash/query_runner/athena.py]: Let's store it as a number and format it in view, in case someone would like to run calculations on it.

**Comment by arikfr** [comment]
@ialeinikov merged. Thank you for your patience.

**Comment by jychen7** [comment]
[Question] I just upgrade to Docker Tag: redash/redash:9.0.0-beta.b42121, however, still didn't see "Cost" in UI?

I can confirm from developer console that "query_cost" is in response body "query_result.data.metadata"
However, the editor UI still shows "Data Scanned xxx MB" without query cost

**Comment by gabrieldutra** [comment]
🤔, just noticed this was merged only with backend code. @arikfr, is that so, or should we add the Metadata in query source page?

**Comment by jychen7** [comment]
I create a PR https://github.com/getredash/redash/pull/5098, but waiting for CI

**Pull request merged**